handbook/ops/finance/arr.md

```diff
@@ -0,0 +1,46 @@
+# ARR
+
+## Definitions
+
+See [definitions of ARR, IARR, new IARR, expansion IARR, and bookings](../../sales/index.md#definitions) in the sales handbook.
+
+## Monthly closing process
+
+Every month the finance team closes the books and reports the latest ARR to the team based on calculations in the ARR source of truth, the [ARR tracker spreadsheet](https://docs.google.com/spreadsheets/d/1Ao3Nqw6gH3yAuZtICV3xo35kKKnI9oKXnvPuTQ0Fh9c/edit). This process will be completed within 5 business days of the end of the month.
+
+### New IARR
+
+1. Throughout the month, finance will take any announced sales and add them to the ARR tracker spreadsheet on the Deals tab. Whenever a sale is announced in the #sales Slack channel, before adding it to the ARR tracker spreadsheet, finance will respond to the AE with their assumptions for the following values, and sales must confirm in writing that each of these is correct:
```

Comment:
We should drive towards using Salesforce as the source of truth for the name, IARR and contract dates (instead of Slack, which isn't considered a source of truth for us)

Reply:
```suggestion
1. Throughout the month, finance will take any announced sales and add them to the ARR tracker spreadsheet on the Deals tab. Whenever an opportunity is marked as "Won" in Salesforce (or when a sale is announced in the #sales Slack channel), before adding it to the ARR tracker spreadsheet, finance will respond to or message the AE with the following values pulled in Salesforce, and sales must confirm in writing that each of these is correct:
```

Reply:
Thoughts? Is my SFDC terminology ("Opportunity", "won") correct?

Reply:
Yeah looks good!